policyengine_taxsim/config/variable_mappings.yaml

```diff
@@ -557,6 +690,10 @@ policyengine_to_taxsim:
     variables:
       - state_non_refundable_credit
       - state_refundable_credits
+    special_cases:
+      - ms:
+          implemented: false
+          variable: na_pe
```

Comment:
same question as above

Reply:
v40: state total credits. Need multiple variable `ms_non_refundable_credit + ms_refundable_credits`, `ms_non_refundable_credit` does not exists, for this case, for this case do we also need to use `ms_files_separately` i did not see any ms_non_refundable_credit_indiv/indv though

Reply:
the variable name is `ms_non_refundable_credits` - this does not depend on the filing separately structure 

In general all of the Mississippi state related variables are in the [variables/gov/states/ms](https://github.com/PolicyEngine/policyengine-us/tree/master/policyengine_us/variables/gov/states/ms/tax/income) folder

The same goes for the other states